src/components/screens/AddonScreen/AddonScreen.js

```diff
@@ -221,6 +221,11 @@ export function PureAddonScreen({ data: { allMediumPost }, ...props }) {
           desc="An elegant way to wrap your component stories and change their contextual environment directly and dynamically in Storybook UI!"
           addonUrl="https://github.com/leoyli/addon-contexts"
         />
+        <AddonItem
```

Comment:
Thought about the same and could agree with that. Ultimately I would like this addon to cover design token documentation going beyond css/scss variables. It already does with covering icons files. That's why I went with the "Organize" category. So, I'm not really sure.

Reply:
As a designer I think most folks associate design systems with the styling. So even though the addon does more a bit more than that (awesome!) – I think it would be more discoverable if it was in the styling section.

Reply:
Sounds reasonable. I moved it.